src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -229,6 +230,23 @@ public async Task<ConflictResolution> ResolveConflictsAsync()
 #if DEBUG
                     await DebugVerifyNoErrorsAsync(conflictResolution, _documentsIdsToBeCheckedForConflict).ConfigureAwait(false);
 #endif
+
+                    // Step 5: Rename declaration files
```

Comment:
Phase 5? This feels like a game of [Power Grid](https://en.wikipedia.org/wiki/Power_Grid), doubly so since the instructions don't match the cards. (And if you haven't played Power Grid...we shall have a game night.)

Reply:
It's called `Step` in comments above, so I'll keep that. I have not played Power Grid

Reply:
@ryzngard @dpoeschl Most indeed, we shall have a Power Grid game night.